### ISSUE TITLE
Autocontext - cannot adjust raw display thresholds for training stage 2

### ISSUE THREAD

**Issue by KarishmaVal** [ilastik/ilastik#2281]
## Describe the bug
Unable to adjust lookup table usage for raw image data display. Resulting error message is:

Window title:

    TypeError

Message content:

    bool object is not subscriptable
    Advanced information about this error may be found in the log file: path\ilastik_log.txt)

## Expected behavior
To be able to adjust the threshold like we can in the first training stage.

## To Reproduce
Within the autocontext workflow
1. Add some images in the data selection applet
2. Select some features
3. Make some annotations and live
4. Select features for stage 2
5. Right click on raw input and select 'adjust thresholds'

## Error message/traceback

If applicable, please add tracebacks with formatting, such as

```pytb
ERROR 2020-07-28 16:04:26,001 excepthooks 48822 4465460672 Unhandled exception in thread: 'MainThread'
ERROR 2020-07-28 16:04:26,004 excepthooks 48822 4465460672 Traceback (most recent call last):
  File "/Applications/ilastik-1.4.0b7-OSX.app/Contents/ilastik-release/ilastik-meta/volumina/volumina/widgets/layercontextmenu.py", line 53, in adjust_thresholds_callback
    dlg = GrayscaleLayerDialog(layer, menu.parent())
  File "/Applications/ilastik-1.4.0b7-OSX.app/Contents/ilastik-release/ilastik-meta/volumina/volumina/widgets/layerDialog.py", line 60, in __init__
    self.grayChannelThresholdingWidget.setValue(layer.normalize[0][0], layer.normalize[0][1])
TypeError: 'bool' object is not subscriptable

```

## Desktop (please complete the following information):
 - ilastik version: 1.4.0b7-OSX
 - OS: macOS 10.14.6

**Comment by KarishmaVal** [comment]
@colem96

**Comment by k-dominik** [comment]
this is fixed in latest beta 1.4.0b8
